common/src/main/java/com/redhat/cloud/notifications/models/NotificationStatus.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.models;
+
+public enum NotificationStatus {
+    // The notification failed to be created for processing
+    // This could happen when sending the notification to a different processor and it was rejected before entering a processing phase
+    FAILED_CREATION,
+    // The notification is being processed
+    // A notification was sent to a different processor and we are waiting for it's output
+    PROCESSING,
```

Comment:
`PROCESSING` and `SENT` are very close. Could we only use `SENT` even from Camel and then change is to `SUCCESS` for Camel while it would remain `SENT` for RHOSE (for now)?

Reply:
Okay, sounds good to me.